mac/KeymanEngine4Mac/KeymanEngine4Mac/CoreWrapper/CoreWrapper.m

```diff
@@ -67,6 +64,38 @@ -(void)changeKeyboardWithKmxFilePath:(NSString*) path {
   }
 }
 
+-(CoreKeyboardInfo*) getKeyboardInfoForKmxFile:(NSString*)kmxFile {
+  NSArray *keyArray = [self getKeyArray];
+  CoreKeyboardInfo* info = [[CoreKeyboardInfo alloc] init:kmxFile keyArray: keyArray];
```

Comment:
Is keyArray and its members leaked here?

Reply:
no, by default there is a strong reference to the key array in CoreKeyboardInfo which is dealloced when the keyboard is changed